docs/e2e.md

```diff
@@ -0,0 +1,35 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Running E2E Tests](#running-e2e-tests)
+    - [Reference](#reference)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Running E2E Tests
+
+1. Ensure **operator-sdk** is installed and login to your OpenShift cluster as an admin user.
+
+1. If the namespace **common-service-operator** exists, delete it. Do this to make sure the namespace is clean.
+
+    ```bash
+    oc delete namespace common-service-operator
```

Comment:
@horis233 Can you please add another option to enable e2e test using https://github.com/kubernetes-sigs/kind , the cs operator should also work with vanila kubernetes.

You can address this comment in another patch.